views/js/controller/Tools/pauseActiveExecutions.js

```diff
@@ -0,0 +1,22 @@
+define(['jquery', 'helpers', 'util/url', 'ui/feedback'], function($, helpers, url, feedback){
```

Comment:
You should add the license header.
Also the `'use strict';` instruction is missing.